Cargo.toml

```diff
@@ -139,5 +139,5 @@ sysinfo = "0.30.10"
 systemstat = "0.2.3"
 
 
-[profile.dev]
-debug = 0 # Speed up compilation time and not necessary.
+# [profile.dev]
+# debug = 0 # Speed up compilation time and not necessary.
```

Comment:
Are we keeping this profile or removing it? If removing, I'd tend to delete and not comment out personally.